sfe.bll/Exceptions/UserNotFoundException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace sfe.bll
```

Comment:
The exceptions are Ok, but I notice the namespace doesn't match with the path. I believe sfe.bll.exceptions could be better in order to group all exceptions in this layer